common/functions/src/scalars/strings/regexp_like.rs

```diff
@@ -100,6 +100,10 @@ impl Function for RegexpLikeFunction {
             mt,
         )?))
     }
+
+    fn passthrough_constant(&self) -> bool {
```

Comment:
I think it's incorrect.

`passthrough_constant` only works when all arguments are constant. So it's ok to `check_get` ConstColumn.

Reply:
Thanks, fixed.  And I also have simplified some of the code for regexp_like function.